static_src/components/usage_and_limits.jsx

```diff
@@ -20,77 +22,86 @@ export default class UsageAndLimits extends React.Component {
-          Using { this.getStat('disk') }  MB
-          { helpText }
-        </span>
+    <div style={{ marginTop: '1rem' }}>
+      <div className={ this.styler('panel-column') }>
+        <ResourceUsage title="Instance disk"
+          amountUsed={ this.getStat('disk') }
+          amountTotal={ this.getStat('disk_quota') }
+          />
       </div>
+      <div className={ this.styler('panel-column') } style={{ textAlign: 'left'}}>
+        <ResourceUsage title="Instance disk"
+          amountTotal={ this.getStat('disk_quota') }
+        />
+      </div>
+    </div>
     );
   }
 
   get memory() {
-    let helpText = <span></span>;
-    if (this.props.app.state.toUpperCase() === 'STOPPED') {
-      helpText = <em> Stopped apps do not use memory.</em>;
-    }
     return (
-      <div>
-        <h5 className={ this.styler('panel-column', 'panel-column-less') }>
-          Memory
-        </h5>
-        <span className={ this.styler('panel-column') }>
-          { this.getStat('mem_quota') } MB
-        </span>
-        <span className={ this.styler('panel-column') }
-            style={{ textAlign: 'left' }}>
-          Using { this.getStat('mem') }  MB.
-          { helpText }
-        </span>
+    <div>
+      <div className={ this.styler('panel-column') }>
+        <ResourceUsage title="Instance memory"
+          amountUsed={ this.getStat('mem') }
+          amountTotal={ this.getStat('mem_quota') }
+        />
       </div>
+      <div className={ this.styler('panel-column') } style={{ textAlign: 'left'}}>
+        <ResourceUsage title="Instance memory"
+          amountTotal={ this.getStat('mem_quota') }
+        />
+      </div>
+    </div>
     );
   }
 
-  // TODO: remove this presentational span
-  get appState() {
+  get totalDisk() {
+    // TODO get space quota
     return (
-      <div>
-        <h5 className={ this.styler('panel-column', 'panel-column-less') }>
-          App state
-        </h5>
-        <span className={ this.styler('panel-column') }>
-          RUNNING
+    <div style={{ marginTop: '1rem' }}>
+      <ResourceUsage title="Total disk"
+        amountUsed={ this.getStat('disk') * this.props.app.running_instances }
+        amountTotal={ this.getStat('disk_quota') }
+      />
+    </div>
+    );
+  }
+
+  get totalMemory() {
+    return (
+    <div>
+      <ResourceUsage title="Total memory"
+        amountUsed={ this.getStat('mem') * this.props.app.running_instances }
+        amountTotal={ this.props.quota.memory_limit * 100000 }
+      />
+    </div>
+    );
+  }
+
+  get scale() {
+    return (
+      <div style={{ textAlign: 'center' }}>
+        <h5>App scale</h5>
+        <span className={ this.styler('stat-primary')}>
+          { this.props.app.running_instances }X
         </span>
-        <span className={ this.styler('panel-column') }></span>
+        <br />
+        <h5 style={{ fontSize: '0.75rem', maxWidth: '5rem', display: 'inline-block', marginTop: '3rem' }}>
```

Comment:
Even if it's not being used anywhere else, putting it in `cg-style` allows things like linting, use of constant variables (like padding or common font sizes), makes search/replace easier.